php/class-fm-bylines.php

```diff
@@ -634,33 +634,34 @@ public function get_the_byline_meta( $field, $byline_id = null ) {
 				$byline_ids = $this->get_byline_ids();
 				$byline_id = reset( $byline_ids );
 			}
+			if ( ! empty( $byline_id ) ) {
```

Comment:
It's possible this could still be entry. The above code simply tries to set an id if one is not passed.